tsconfig.json

```diff
@@ -45,5 +46,8 @@
 		"./tests/unit/all.ts",
 		"./tests/unit/basicCommonJsLoading.ts",
 		"./tests/unit/require.ts"
-	]
+	],
+	"atom": {
```

Comment:
Bah! Don't let editors add this poop! :frowning: (I'm not really that mad) I am still mad at atom-typescript for doing this.  Silly silly.